pynetgear/__init__.py

```diff
@@ -271,6 +272,71 @@ def tofloats(lst): return (float(t) for t in lst)
             return None
 
         return {t.tag: parse_text(t.text) for t in node}
+        
+    def config_start(self):
+        """
+        Start a configuration session.
+        For managing router admin functionality (ie allowing/blocking devices)
+        """
+        _LOGGER.info("Config start")
+
+        message = "<NewSessionID>{session_id}</NewSessionID>".format(session_id=SESSION_ID)
+
+        success, _ = self._make_request(
+            SERVICE_DEVICE_CONFIG, "ConfigurationStarted", message)
+        
+        self.config_started = success
+        return success
+    
+    def config_finish(self):
+        """
+        End of a configuration session.
+        Tells the router we're done managing admin functionality.
+        """
+        _LOGGER.info("Config finish")
+        if not self.config_started:
+        	return True
+
+        message = "<NewStatus>ChangesApplied</NewStatus>"
+
+        success, _ = self._make_request(
+            SERVICE_DEVICE_CONFIG, "ConfigurationFinished", message)
+        
+        self.config_started = not success
+        return success
+    
+    def allow_block_device(self, mac_addr, device_status='Block'):
```

Comment:
for device_status please define 2 constants BLOCK and ALLOW so users can use them too directly.

Reply:
Fixed :)